1.9/administration/installing/custom/advanced.md

```diff
@@ -82,10 +82,11 @@ The DC/OS installation creates these folders:
     - 8.8.4.4
     - 8.8.8.8
     use_proxy: 'true'
-    http_proxy: http://<your_http_proxy>/
-    https_proxy: https://<your_https_proxy>/
+    http_proxy: http://<proxy_host>:<http_proxy_port>
```

Comment:
@vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.